### PR TITLE
move access logs to access.log

### PR DESCRIPTION
**Pull request by jrwren**

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1849/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1850/
Test PASSed.

File: cmd/charmd/main.go [hunk 1]
```diff
@@ -116,9 +116,18 @@ func serve(confPath string) error {
 	if err != nil {
 		return errgo.Notef(err, "cannot create new server at %q", conf.APIAddr)
 	}
-
+	handler := server.(http.Handler)
```
Comment (urosj) [inline cmd/charmd/main.go:119]: s/handler/server, bike-shedding here :)

**Comment by urosj** [comment]
👍

File: cmd/charmd/main.go (hunk 1 above)
Comment (jrwren) [inline cmd/charmd/main.go:119]: I wanted to do that, but NewServer returns an HTTPCloser and so its a different type than CombinedLoggingHandler returns and so I needed to use a different variable. I took a look a net/http to make sure it doesn't call close and I found nothing so I'm pretty sure we only have this HTTPCloser for testing.

**Comment by jrwren** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Pull request merged**